src/api/api-clients/restful-booker-platform/rooms-api-client.ts

```
@@ -10,41 +11,62 @@ import { AxiosResponse } from "axios";
+    room_id: number,
+    params?: object,
+  ): Promise<AxiosResponse> {
     const response = await this.apiClient.id.deleteRoom(room_id, params);
     return response;
   }
 
-  async createRoom(data: Room, params?: object): Promise<AxiosResponse> {
+  async deleteRoom(
+    room_id: number,
+    params?: object,
+  ): Promise<AxiosResponse | void> {
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)